testing/netdata/APKBUILD

```diff
@@ -19,7 +19,7 @@ builddir="$srcdir"/$pkgname-$pkgver
 
 prepare() {
 	default_prepare
-	update_config_guess
+	update_config_guess || true
```

Comment:
You don't need this. If `update_config_guess` fails, it means the files don't need updating and the call should be removed. In that case, you could remove the whole `prepare()` block.

Reply:
@fabled done, thanks.